src/status_im2/subs/communities.cljs

```diff
@@ -245,10 +246,21 @@
                  :unread-messages? (pos? unviewed-messages-count)
                  :mentions-count   (or unviewed-mentions-count 0)
                  :locked?          (or (not joined) (not can-post?))
+                 :muted?           muted
                  :id               id})))
     {}
     chats)))
 
+(re-frame/reg-sub
+ :communities/channel-by-id
```

Comment:
1. When the name of something ends with `by-id` it means the output is a map, often indexing or grouping by id. But such is not the case of this subscription @ibrkhalil. The output is a single channel or perhaps I got things wrong? The name could be `:communities/channel` which implies a single channel is being returned.

2. This layer-3 sub should use a quick unit test. It would've helped me a lot when reviewing this sub because it'd be obvious what it does and returns.

Reply:
I thought I'd need this sub, But I ended up deleting it because I don't need it anymore